src/test/scala/mesosphere/marathon/core/appinfo/TaskCountsTest.scala

```diff
@@ -0,0 +1,132 @@
+package mesosphere.marathon.core.appinfo
+
+import mesosphere.marathon.health.Health
+import mesosphere.marathon.state.Timestamp
+import mesosphere.marathon.{ MarathonSpec, Protos }
+import mesosphere.util.Mockito
+import org.apache.mesos.{ Protos => mesos }
+import org.scalatest.{ GivenWhenThen, Matchers }
+
+class TaskCountsTest extends MarathonSpec with GivenWhenThen with Mockito with Matchers {
```

Comment:
I'd like to see one _more complex_ test with not only one task in it but several – maybe `TaskCounts` counts only the first item in the given `appTasks` ;)